Indexing/TermSharing.cpp

```diff
@@ -216,11 +218,11 @@ AtomicSort* TermSharing::insert(AtomicSort* sort)
   ASS(!sort->isSpecial());
   ASS(sort->isSort());
 
-  // cannot use TC_TERM_SHARING
+  // cannot use TERM_SHARING
```

Comment:
Is this still true? Could we fix it now?

Reply:
I actually didn't look into this, because i just did a semi-automated refactoring for replacing `TimeCounter` vs `TIME_TRACE`. 
We can definitely use `TERM_SHARING` here with the new impl. The question is whether we want to. I think as probably noone is using the time statistics right now we can just remove the comment and if someone needs to know some specific details they can just change the macro call.